src/components/ProjectListItemCard/ProjectListItem.graphql

```diff
@@ -28,7 +28,12 @@ fragment ProjectListItem on Project {
       total
     }
   }
-  ... on TranslationProject {
+  ... on MomentumProject {
+    engagements {
+      total
+    }
+  }
+  ... on MultiplicationProject {
```

Comment:
There's no need to intersect this on concretes. The Project interface has this field.
```gql
...TogglePin
engagements { total }
```